src/DetectAdblock.js

```diff
@@ -15,16 +15,10 @@ export default class DetectAdblock {
   }
 
   startChecking(count = defaultCount) {
-    if (!process.env.IS_BROWSER) {
-      return;
-    }
+    if (typeof process !== 'undefined') if (!process.env.IS_BROWSER) return;
```

Comment:
Basically I will change the `IS_BROWSER` definition from `process.env.IS_BROWSER`  to `typeof window !== 'undefined'`. `process.env` is a variable which can be set in `webpack` and I agree it isn't necessary to set it.